force_wfmanager/ui/review/tests/test_plot.py

```diff
@@ -1,291 +1,38 @@
-import mock
-import time
 import unittest
 import warnings
 
 from chaco.api import Plot as ChacoPlot
-from chaco.api import ColormappedScatterPlot, ScatterPlot
+from chaco.api import BaseXYPlot
+from chaco.api import ColormappedScatterPlot
 from chaco.abstract_colormap import AbstractColormap
 
-from force_wfmanager.model.analysis_model import AnalysisModel
-from force_wfmanager.ui.review.plot import BasePlot, Plot
+from force_wfmanager.ui.review.base_plot import BasePlot
+from force_wfmanager.ui.review.color_plot import ColorPlot
 from traits.api import push_exception_handler, TraitError
 
-push_exception_handler(reraise_exceptions=True)
-
-
-class TestAnyPlot(object):
```

Comment:
This class provides unit test methods for `TestBasePlot` and `TestColorPlot` classes - I've moved it into a separate file and renamed it `BasePlotTestCase`.